usr/sbin/p11sak/p11sak.c

```diff
@@ -8,9 +8,9 @@
  * https://opensource.org/licenses/cpl1.0.php
  */
 
+#include <stdio.h>
 #include <stdlib.h>
```

Comment:
Mixing changes in a single commit.....  I would separate this, and sqaush this change with the original commit where you introduce that code

Reply:
Done.